src/System.Security.Cryptography.Xml/src/Resources/SR.Designer.cs

```diff
@@ -0,0 +1,838 @@
+﻿//------------------------------------------------------------------------------
```

Comment:
The designer file is automatically generated (as SR.cs) in the obj directory during build (and added as a compile source).  The `Resources` directory should only contain one file, `Strings.resx`.  Then everything else happens by magic.  (In VS you'll have a lot of errors reported until you actually build from the command line, its soft compile seems to skip the SR.cs generation step)